sw/device/tests/rstmgr_alert_info_test.c

```diff
@@ -701,6 +701,12 @@ static void init_expected_cause(void) {
       .alert_info.alert_cause[kTopEarlgreyAlertIdI2c0FatalFault] = 1;
   kExpectedInfo[kRound3]
       .alert_info.alert_cause[kTopEarlgreyAlertIdSpiHost0FatalFault] = 1;
+
+  if (kBootStage == kBootStageOwner &&
```

Comment:
On Round 3 all the alerts are enabled, after the reset is caused by the alert handler escalation, a `kTopEarlgreyAlertIdFlashCtrlFatalErr` happens during the boot, only with FGPA + ROM_EXT. 
This doesn't happen on Silicon, probably because the Creator and Owner secrets are there, but I'm not sure why it happens on the FPGA. 
@cfrantz Could you check that this is not a bug with ROM_EXT?